src/audio/pipeline.c

```diff
@@ -121,9 +121,9 @@ int pipeline_connect(struct comp_dev *comp, struct comp_buffer *buffer,
 	uint32_t flags;
 
 	if (dir == PPL_CONN_DIR_COMP_TO_BUFFER)
-		comp_info(comp, "connect buffer %d as sink", buffer->id);
+		comp_info_throttle(comp, "connect buffer %d as sink", buffer->id);
 	else
-		comp_info(comp, "connect buffer %d as source", buffer->id);
+		comp_info_throttle(comp, "connect buffer %d as source", buffer->id);
```

Comment:
_rate_limited() ?

Reply:
This one should stay as comp_info as it's only emitted at topology loading.